tests/functional/tests/rgw/test_rgws.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+
+
+class TestRGWs(object):
+
+    @pytest.mark.no_docker
+    def test_ceph_rgw_package_is_installed(self, node, Package):
+        assert Package("radosgw").is_installed
+
+
+    def test_rgw_services_are_running(self, node, Service):
+        # TODO: figure out way to paramaterize node['rgws'] for this test
+        for rgw in node["rgws"]:
+            assert Service("ceph-radosgw@%s" % rgw).is_running
```

Comment:
so this becomes:

    ansible_vars = Ansible.get_variables()
    hostname = ansible_vars['hostname']
    assert Service("ceph-radosgw@%s" % hostname).is_running

I don't know what the right key in ansible_vars is for the `hostname`, but that should be easy to find out I think?